independent-publisher-connectors/Workable/settings.json

```diff
@@ -0,0 +1,9 @@
+{
+  "connectorId": "shared_workable-5f4c0b1d44968dd354-5fb23ccf32fe974138",
```

Comment:
@DavidBreathe settings.json file is not required from IP connectors. please remove this file.